File: Phase05/Phase05/Phase05/QueryKeeper.cs
using System.Collections.Generic;

namespace Phase05
{
    public class QueryKeeper
    {
        public HashSet<string> _plusContain { get;}
        public HashSet<string> _minusContain { get;}
        public HashSet<string> _withoutSignContain { get;}
        public QueryKeeper(HashSet<string> plusContain, HashSet<string> minusContain, HashSet<string> withoutSignContain)
        {
            _plusContain = plusContain;
            _minusContain = minusContain;
            _withoutSignContain = withoutSignContain;
        }

        
        
    }
}

File: Phase03/src/QueryKeeper.java
package Phase02;

import java.util.HashSet;
import java.util.Set;

public class QueryKeeper {
    private final Set<String> plusContain;
    private final Set<String> minusContain;
    private final Set<String> withOutSign;

    public QueryKeeper(Set<String> plusContain, Set<String> minusContain, Set<String> withOutSign) {
        this.plusContain = plusContain;
        this.minusContain = minusContain;
        this.withOutSign = withOutSign;
    }

    public Set<String> getPlusContain() {
        return plusContain;
    }

    public Set<String> getMinusContain() {
        return minusContain;
    }

    public Set<String> getWithOutSign() {
        return withOutSign;
    }


}
